packages/kotti-ui/source/kotti-filter/components/ButtonLink.vue

```diff
@@ -8,6 +8,7 @@
 </template>
 
 <script lang="ts">
+import { Yoco } from '@3yourmind/yoco'
```

Comment:
How did this not error before?

Reply:
Should it? I don't see what's wrong here or am I missing something? This import is in various components.

Reply:
On my diff this was the only line you added. Either there would’ve been a usage before or ESLint should complain that the import is unused. Neither of which happened which is why I’m confused